pkg/kubectl/cmd/get/get.go

```diff
@@ -532,11 +532,10 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 			w.Flush()
 			w.SetRememberedWidths(nil)
 
-			// TODO: this doesn't belong here
 			// add linebreak between resource groups (if there is more than one)
 			// skip linebreak above first resource group
 			if lastMapping != nil && !o.NoHeaders {
-				fmt.Fprintln(o.ErrOut)
+				fmt.Fprintln(w)
```

Comment:
what does this do for other output types like jsonpath or yaml?

Reply:
@liggitt if the output type is jsonpath or yaml, [the `IsHumanReadablePrinter ` flag will be false](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/get/get.go#L229), which means that this code change won't affected printing jsonpath or yaml because we will return much earlier in `Run`: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/get/get.go#L477

Reply:
It seems we have tests covering jsonpath on multiple resources:

https://github.com/kubernetes/kubernetes/blob/3d4124f2e083591c98e8a874760011781d3ee15c/pkg/kubectl/cmd/get/get_test.go#L952-L953

And that CI is green sort of proves @taragu's statement.

Reply:
But it'd be nice to know the intension that why in the beginning the blank line was appended to stderr instead of stdout.

Reply:
@Huang-Wei I checked git blame and this change (printing blank lines to stderr) was added by @juanvallejo in [this commit](https://github.com/kubernetes/kubernetes/pull/62881/files). @juanvallejo is there a particular reason why we are currently appending blank lines to stderr?